examples/prom/elliptic_eigenproblem_global_rom.cpp

```diff
@@ -585,6 +611,9 @@ int main(int argc, char *argv[])
         // Calculate errors of eigenvectors
         ostringstream mode_name_fom;
         Vector mode_fom(numRowRB);
+        const double eigenvalue_threshold = max(1e-6,
```

Comment:
Should the hard-coded values 1e-6 and 0.02 be input parameters?

Reply:
New input parameter `eig_tol` is introduced.